renderdoc/driver/gl/official/cgl.h

```diff
@@ -38,8 +38,13 @@ typedef char     GLchar;
 #endif
 #if !defined(GL_ARB_shader_objects)
 typedef char     GLcharARB;
+typedef char GLcharARB;
+#if defined(RENDERDOC_PLATFORM_SWITCH)
```

Comment:
Why do you need to make this change to the OS X cgl headers? These are official headers and shouldn't be changed.

Reply:
I'll double-check with the team (this one is from before my time).  I can revert it.